orderbook/src/orderbook.rs

```diff
@@ -172,7 +173,7 @@ impl Orderbook {
         Ok(orders)
     }
 
-    pub async fn get_order(&self, uid: &OrderUid) -> Result<Option<Order>> {
+    pub async fn get_order_for_uid(&self, uid: &OrderUid) -> Result<Option<Order>> {
```

Comment:
uber-naming-nit: I kind of prefer just `get_order` here as it takes a single UID as a parameter.

Reply:
No strong feelings here. If you feel like this makes the code easier to read, then leave it in.